website/docs/docs/deploy/job-notifications.md

```diff
@@ -52,16 +52,28 @@ If there has been a change in user roles or Slack permissions where you no longe
 ### Prerequisites 
 - You must be an administrator of the Slack workspace. 
 - You must be an account admin to configure Slack notifications in dbt Cloud. For more details, refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users).
-- Make sure the notification channel (where you want to receive alerts) is a public channel. The integration only supports public channels in the Slack workspace. 
+- The integration only supports _public_ channels in the Slack workspace. 
 
 ### Set up the Slack integration
 
-1. From the gear menu, select **Profile settings**. On your **User profile** page, scroll to the **Linked accounts** section.
-1. In the **Linked accounts** section, find the Slack application and click **Link**.
+1. From the gear menu, select **Account settings** and then select **Integrations** from the left navigation pane. 
+1. Locate the **OAuth** section with the Slack application and click **Link**.
    <Lightbox src="/img/docs/dbt-cloud/Link-your-Slack-Profile.png" width="75%" title="Link for the Slack app"/>
```

Comment:
non-blocking: maybe as a fast-follow, update the screenshot so the Red Arrow does not cross any text

Reply:
Replaced it!